src/components/PlatformPayButton.tsx

```diff
@@ -144,7 +147,6 @@ export function PlatformPayButton({
   return (
     <TouchableOpacity
       disabled={disabled}
-      activeOpacity={disabled ? 0.3 : 1}
```

Comment:
Not sure why there is an `activeOpacity` from my POV there is no need, but you may have an explanation ;)